modules/node_modules/@ncigdc/components/DownloadTableToTsvButton.js

```diff
@@ -38,7 +38,7 @@ const DownloadTableToTsvButton = ({
           return reduce(tds, (acc, td) => {
             const markedForTsv = td.querySelector('.for-tsv-export');
             const exportText = markedForTsv ? markedForTsv.innerText : td.innerText;
-            const joinedText = exportText.trim().replace(/\s*/, ' ').split('\n').join(',');
```

Comment:
it should be catching the `&nbsp;` there is likely just another whitespace character before it :D just needs the global flag on the regex :)